eksctl/cluster-config.yaml

```diff
@@ -8,21 +8,22 @@ metadata:
 managedNodeGroups:
   - name: helper1
     labels: { releaseName: h1 }
-    instanceType: c6id.8xlarge
+    instanceType: c6id.12xlarge
     desiredCapacity: 1
   - name: helper2
     labels: { releaseName: h2 }
-    instanceType: c6id.8xlarge
+    instanceType: c6id.12xlarge
     desiredCapacity: 1
   - name: helper3
     labels: { releaseName: h3 }
-    instanceType: c6id.8xlarge
+    instanceType: c6id.12xlarge
     desiredCapacity: 1
   - name: rc
     labels: { releaseName: rc }
     instanceType: c6id.8xlarge
     volumeSize: 2000
     desiredCapacity: 1
+availabilityZones: ['us-east-1a']
```

Comment:
why they are all in the same AZ?

Reply:
2 things: 
1. its better for networking speed
2. 1a is the larger region, with this constrain we avoid getting hosts in one of the smaller regions.